src/documentation/0045-node-folders/index.md

```diff
@@ -109,6 +111,19 @@ fs.rmdir(dir, { recursive: true }, (err) => {
 });
 ```
 
+> **NOTE:** In Node `v16.x` the option `recursive` is **deprecated** for `fs.rmdir` of callback API, instead use the below code to delete folders that have content in them:
+
+```js
+const fs = require('fs/promises')
+
+try {
+    fs.rm(dir, { recursive: true, force: true })
```

Comment:
Since `fs.rm` is a promise here, it'll go to `UnhandledPromiseRejectionWarning` instead of getting caught into the catch block.

You can use the normal `fs` module here, to maintain consistency with the snippet before this

Reply:
nice catch, thanks 😅